iac-extensions/kubernetes/src/test/resources/checks/MemoryRequestCheck/memory_request_deployment.yaml

```diff
@@ -0,0 +1,54 @@
+apiVersion: apps/v1
+kind: Deployment
```

Comment:
Tests for other kinds are missing. See `CommandExecutionCheckTest` for inspiration.